Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/grpc/grpc-swift.git", from: "1.19.1"),
         .package(url: "https://github.com/stephencelis/SQLite.swift.git", from: "0.14.1"),
-        .package(url: "https://github.com/zcash-hackworks/zcash-light-client-ffi", exact: "0.7.4")
+        .package(url: "https://github.com/zcash-hackworks/zcash-light-client-ffi", from: "0.8.0")
```

Comment:
Because Swift SemVer is annoying, keep this as before:
```suggestion
        .package(url: "https://github.com/zcash-hackworks/zcash-light-client-ffi", exact: "0.8.0")
```